src/reports/assessment-report-model-builder.ts

```diff
@@ -6,7 +6,8 @@ import { Assessment } from 'assessments/types/iassessment';
 import { ManualTestStatus } from 'common/types/manual-test-status';
 import { AssessmentData, AssessmentStoreData, TestStepInstance } from 'common/types/store-data/assessment-result-data';
 import { TabStoreData } from 'common/types/store-data/tab-store-data';
-import * as _ from 'lodash';
+import { keys } from 'lodash';
+import { find } from 'office-ui-fabric-react';
```

Comment:
I'm curious why the change here uses `find` from `office-ui-fabric-react` instead of `lodash`?

Reply:
Good catch! Thank for finding this.